routing/routing.xml

```diff
@@ -614,12 +624,22 @@
 			<select value="1.3" t="highway" v="steps"/>
 			<select value="3" t="route"   v="ferry"/>
 			<select value="5" t="bicycle" v="dismount"/>
-			<select value="22" t="bicycle" v="designated"/>
-			<if param="avoid_unpaved">
-				<select value="4" t="highway" v="track"/>
-				<select value="4" t="highway" v="footway"/>
-				<select value="3" t="highway" v="path"/>
-			</if>
+			<!-- Line above: I now think it would be a better solution
```

Comment:
Yes, you are absolutely right. Good point. The problem exists in driving style "speed" (not in the other two style). There is a smooth solutions for this: 

We just move the fixed kmh value for bicycle=designated a few lines further down, into the exceptions for the driving style "speed". I have already put this into the pull request. I have also deleted the remark about elevators (thanks for changing that in the map rendering).